platform/mellanox/sonic-bfb-installer

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+command_name="sonic-bfb-installer"
+usage(){
+    echo "Syntax: $command_name -b|--bfb <BFB_Image_Path> --rshim|-r <rshim1,..rshimN> --verbose|-v --config|-c <Options> --help|h"
+    echo "Arguments:"
+    echo "-b			Provide custom path for bfb image"
+    echo "-r			Install only on DPUs connected to rshim interfaces provided, mention all if installation is requried on all connected DPUs"
+    echo "-v			Verbose installation result output"
+    echo "-c			Config file"
+    echo "-h		        Help"
+}
+
+bfb_install_call(){
+    #Example:sudo bfb-install -b <full path to image> -r rshim<id>
+    local result_file=$(mktemp "/tmp/result_file.XXXXX")
+    local cmd="sudo ./bfb-install -b $2 -r $1 $appendix"
+    echo "Installing bfb image on DPU connected to $1 using $cmd"
+    local indicator="$1:"
+    eval "$cmd" > "$result_file" 2>&1 > >(while IFS= read -r line; do echo "$indicator $line"; done > "$result_file")
```

Comment:
What is this doing? From what i understand stdout & stderr are already writtem to the $result_file

`> >(while IFS= read -r line; do echo "$indicator $line"; done > "$result_file")`

Reply:
Since we are executing the same command for each dpu separately, the indicator is added at the start of each line to differentiate between the execution between multiple dpus